inc/issue.class.php

```diff
@@ -521,7 +522,16 @@ public static function giveItem($itemtype, $option_id, $data, $num) {
       switch ("$table.$field") {
          case "glpi_plugin_formcreator_issues.name":
             $name = $data[$num][0]['name'];
-            return "<a href='".FORMCREATOR_ROOTDOC."/front/issue.form.php?id=".$id."&sub_itemtype=".$data['raw']['sub_itemtype']."'>$name</a>";
+            $ticket->getFromDB($data[$num][0]['id']);
```

Comment:
if the issue is from a form without ticket or with several tickets, the ID will be a PluginFormcreatorFormAnswer ID. Then you might show a qtip for the unexpected ticket.

I don't see for now a better solution than checking the 1st character if display_id (it is 't' for a ticket, or 'f' for form answer). If the character is 't' then you can show the qtip. In all other cases you should use "N/A" for the qtip, or disable it.